lms/services/assignment.py

```diff
@@ -193,6 +194,15 @@ def upsert_assignment_groupings(
             )
         )
 
+    def get_assignment_groupings(self, assignment: Assignment) -> list[Grouping]:
```

Comment:
We need all groupings to find annotations in the course group, the sections and/or groups.